prep-css-box-model/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+}
```

Comment:
Hi Derek, your box model properties should go inside of this `button` ruleset so that there is margin, padding, and a border applied to the buttons.